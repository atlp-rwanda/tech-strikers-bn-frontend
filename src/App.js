import React from "react";

const App = () => (
  <div>Hello! This Barefoot Nomad created from Scratch. WOW</div>
);

export default App;
