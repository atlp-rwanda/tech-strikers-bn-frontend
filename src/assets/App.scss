* {
  margin: 0px;
  padding: 0px;
}
.modal {
  width: 105%;
}
.modal-content {
  width: 80%;
}
.login-btn,
.signup-btn {
  background-color: rgba(255, 255, 255, 0);
  color: #fff;
  border: 0px;
}
.login-btn::after,
.signup-btn::after {
  border: none;
}
.login-submit {
  width: 100%;
  height: 40px;
  padding: 0px;
  margin-top: 15px;
}
.reset-pass-link {
  text-decoration: none;
}
.icon-login-div {
  margin-left: 45%;
}
.authIcon {
  display: block;
  width: 20%;
}
.close {
  width: 20%;
  border: none;
  margin-left: 85%;
  -webkit-appearance: button;
}
.starIcon {
  width: 8%;
}
.locationIcon {
  width: 10%;
  height: 10%;
  margin: 0px;
  padding: 0px;
}
.cardBox {
  width: 85%;
  margin-right: 30%;
}
.cardContent {
  padding: 7%;
}
.row.content {
  height: 450px;
}

.sidenav {
  padding-top: 20px;
  background-color: #f1f1f1;
  height: 100%;
}

/* Set black background color, white text and some padding */
footer {
  background-color: #555;
  color: white;
  padding: 15px;
}

/* On small screens, set height to 'auto' for sidenav and grid */
@media screen and (max-width: 767px) {
  .sidenav {
    height: auto;
    padding: 15px;
  }
  .row.content {
    height: auto;
  }
}
.error {
  border: 2px solid #ff6565;
}
