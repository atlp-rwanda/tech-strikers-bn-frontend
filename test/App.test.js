/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import App from "../src/App";

it("snapshot test", () => {
  const appTest = renderer.create(<App />).toJSON();
  expect(appTest).toMatchSnapshot();
});
