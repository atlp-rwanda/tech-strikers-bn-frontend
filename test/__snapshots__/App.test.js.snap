// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div>
  Hello! This Barefoot Nomad created from Scratch. WOW
</div>
`;
